harvest/plugin/yahoo.py

```diff
@@ -1,28 +0,0 @@
-import pandas as pd
```

Comment:
We don't use this and when I it seems that the `YahooEarningsCalendar` is broken. When I call the `get_earnings_of` function, it always errors out.